src/test/java/core/basesyntax/service/read/FileReaderTest.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.service.read;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileReaderTest {
+    private static final String VALID_FILE_PATH = "src/test/java/resources/testFile.txt";
+    private static final String EMPTY_FILE_PATH = "src/test/java/resources/emptyFile.txt";
+    private static final String NON_EXISTENT_FILE_PATH
+            = "src/test/java/resources/nonExistentFile.txt";
```

Comment:
```suggestion
            = "invalid/path/to/file.txt
```
you can make it even more obvious